//package com.sincetimes.redwar.game.comnunicate.protocol;

message RseUpdateStarsBookmarks
{
	optional int32 ret = 1;
}